cmd/px-node-wiper/px-node-wiper.sh

```diff
@@ -79,8 +79,8 @@ fi
 
 
 # Remove systemd service (if any)
-run_with_nsenter "systemctl stop portworx" true
-run_with_nsenter "systemctl disable portworx" true
+systemctl stop portworx || true
+systemctl disable portworx || true
 run_with_nsenter "rm -rf /etc/systemd/system/portworx.service" false
 run_with_nsenter "rm -rf /etc/systemd/system/portworx-reboot.service" false
```

Comment:
Would be good to add 2 more service-files (newly introduced px proxy-logger):
* `/etc/systemd/system/portworx-output.service` and
* `/etc/systemd/system/portworx.socket`

Alternatively, just wipe all `/etc/systemd/system/*portworx*` -files

Reply:
Done. Simplified with just `rm -rf /etc/systemd/system/*portworx*`